distributed/scheduler.py

```diff
@@ -1849,32 +1831,6 @@ def decide_worker(self, ts: TaskState) -> WorkerState | None:
 
         return ws
 
-    def set_duration_estimate(self, ts: TaskState, ws: WorkerState) -> float:
```

Comment:
This is just a cut-paste out of the transitions section, as this is not a transition-specific method